app/src/main/java/in/testpress/testpress/authenticator/LoginActivity.java

```diff
@@ -484,6 +490,8 @@ protected void finishLogin() {
         sharedPreferences.edit().putBoolean(GCMPreference.SENT_TOKEN_TO_SERVER, false).apply();
         CommonUtils.registerDevice(this, testpressService);
         final Account account = new Account(username, APPLICATION_ID);
+        deleteDownloadedVideos();
```

Comment:
When I read it like story it says deleteDownloadedVideos but when I go inside this method it is deleting based on some condition.